cloudwatch-integ/kvsWebRTCClientViewerCloudwatch.cpp

```diff
@@ -0,0 +1,374 @@
+#include <aws/core/Aws.h>
+#include "../samples/Samples.h"
+#include "Cloudwatch.h"
+
+extern PSampleConfiguration gSampleConfiguration;
+
+#ifdef ENABLE_DATA_CHANNEL
+// onMessage callback for a message received by the viewer on a data channel
+VOID dataChannelOnMessageCallback(UINT64 customData, PRtcDataChannel pDataChannel, BOOL isBinary, PBYTE pMessage, UINT32 pMessageLen)
+{
+    UNUSED_PARAM(customData);
+    UNUSED_PARAM(pDataChannel);
+    if (isBinary) {
+        DLOGI("DataChannel Binary Message");
+    } else {
+        DLOGI("DataChannel String Message: %.*s", pMessageLen, pMessage);
+    }
+}
+
+// onOpen callback for the onOpen event of a viewer created data channel
+VOID dataChannelOnOpenCallback(UINT64 customData, PRtcDataChannel pDataChannel)
+{
```

Comment:
Can we move these to `lib/DataChannelHandling.c`?

Reply:
I am inclined to letting this be in the samples since viewer is creating the data channel and setting the operation on incoming data. Let me know if you see a strong reason to move this into lib/

Reply:
I suggested that because I thought the purpose to have lib/ was to modularize Common.c but I see that it is to retain the pieces used by the customer as well